operator/tests/e2e-v2/centralized-configuration-injected-ref/00-redpanda-cluster.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Just realized this is here... so not sure if you're aware, but we just removed the entirety of the `e2e-v2` tests and ported them to the go/gherkin-based `acceptance` folder. Personally I'm fine merging this as is once the rest of the PR gets signed off and then I can help port this test over too... but just FYI that in the process of porting the tests I also completely removed the `e2e-v2` tests from CI, so these aren't running currently. If you've run them locally I'm fine with that and will ensure this is all fine and dandy in an acceptance test prior to any release we may do, but not entirely sure how @chrisseto feels about that.

Reply:
See https://github.com/redpanda-data/redpanda-operator/pull/827

Reply:
Merged that (I had run them locally to check behaviour). But the alternatives to kuttl are preferable in terms of time and amenability to debugging, so this is great - thanks!